linetools/lists/linelist.py

```diff
@@ -593,48 +621,46 @@ def all_transitions(self, line):
         if line == 'unknown':
             return self.unknown_line()
         if self.list in ['H2']:
-
             cond = self._data['ion_name'] == line.split('(')[0]
```

Comment:
Note that the self._data['ion_name'] does not exist.  Should be self._data['name'].